### PR TITLE
Completion of LCS implementation in JS issue#2252

### LINKED ISSUE #2252: Longest Common Subsequence
- [ ] C - @NVS16
- [x] C++
- [ ] C# - @Yashank18
- [ ] CoffeeScript
- [ ] Dart - @JainStuti25
- [x] Go
- [x] Java
- [x] JS - @abhu-A-J
- [ ] Kotlin - @rbjoshi1309
- [ ] Php - @thakran14
- [x] Python
- [ ] Readme - @NikitaMasand
- [x] Ruby - @Tahanima
- [ ] Rust
- [ ] TypeScript - @adi44

### PR DESCRIPTION
**Pull request by abhu-A-J**
Fixes #2252 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added implementation of Longest Common Subsequence in JS

#### Languages Used:
- JavaScript 

#### Files Added:
- Added file names Longeest_Common_Subsequence.js inside Longeest_Common_Subsequence folder

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by abhu-A-J** [comment]
@raghav-dalmia Sir, can you review this?

File: Longest_Common_Subsequence/Longest_Common_Subsequence.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+/*
+Problem Statement - Given two sequences, find the length of longest subsequence present in 
+both of them.A subsequence is a sequence that can be derived from another sequence by 
+deleting some or no elements without changing the order of the remaining elements.
+*/
+
+const Longest_Common_Subsequence = (arr1, arr2) => {
+
+    // make an array of arrays of size arr1.length+1 * arr2.length+1
+
```
Comment (asha15) [inline Longest_Common_Subsequence/Longest_Common_Subsequence.js]: remove this line

File: Longest_Common_Subsequence/Longest_Common_Subsequence.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+/*
+Problem Statement - Given two sequences, find the length of longest subsequence present in 
+both of them.A subsequence is a sequence that can be derived from another sequence by 
+deleting some or no elements without changing the order of the remaining elements.
+*/
+
+const Longest_Common_Subsequence = (arr1, arr2) => {
+
+    // make an array of arrays of size arr1.length+1 * arr2.length+1
```
Comment (asha15) [inline Longest_Common_Subsequence/Longest_Common_Subsequence.js]: keep space around operator

File: Longest_Common_Subsequence/Longest_Common_Subsequence.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+    let LCS_def = Array.from({ length: arr1.length + 1 },
+        () => Array.from({ length: arr2.length + 1 },
+            () => 0));
+    
+    // iterate over the array and check for LCS 
+
```
Comment (asha15) [inline Longest_Common_Subsequence/Longest_Common_Subsequence.js]: remove this line

File: Longest_Common_Subsequence/Longest_Common_Subsequence.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+                LCS_def[i][j] = 0;
+            }
+
+            // when the last character of both match, increase length of lcs by 1
+            else if (arr1[i - 1] == arr2[j - 1]) {
+                LCS_def[i][j] = LCS_def[i - 1][j - 1] + 1;
+            }
+                
+            // when the last character is not same, take maximum obtained by adding 
+            // one character to one of the subsequences
```
Comment (asha15) [inline Longest_Common_Subsequence/Longest_Common_Subsequence.js]: make this as multi line commend

**Comment by asha15** [comment]
keep going @abhu-A-J

**Pull request merged**